src/renderer/containers/widgets/createCommunity/createCommunity.test.tsx

```diff
@@ -73,6 +75,7 @@ describe('User', () => {
         const data = input as socketEventData<[string]>
 
         const id = data[0]
+        payloadId = payload(id)
```

Comment:
I believe this variable should be named "payload" instead of "payloadId" as it's a bit confusing when you mock address response